docs/general/post-install/transcoding/hardware-acceleration/index.md

```diff
@@ -148,7 +148,7 @@ Dolby Vision (P5 & P8) to SDR tone-mapping is supported in Jellyfin 10.8 and req
 
 - VPP is prefered if both tone-mapping options are enabled.
 
-- Rockchip RKMPP currently only support HDR10 and HLG tone-mapping.
+- Rockchip RKMPP only support HDR10 and HLG tone-mapping in Jellyfin 10.9 and 10.10.
```

Comment:
```suggestion
- Rockchip RKMPP only support HDR10 and HLG tone-mapping in Jellyfin 10.9 or newer versions
```
or do we plan on removing them soon?

Reply:
but the edit would still be correct then. Just that 10.11 has additional features.
Otherwise it reads that 10.11 would not support it.

Reply:
Well indeed. How about this?
```
Rockchip RKMPP does not support Dolby Vision tone-mapping in Jellyfin 10.9 and 10.10.
```

This way there is no need to modify the docs when 10.11 is released.